plugins/catalog-react/report-alpha.api.md

```diff
@@ -74,6 +74,23 @@ export const catalogReactTranslationRef: TranslationRef<
   }
 >;
 
+// @alpha (undocumented)
+export type ContextMenuItemComponent = (
+  props: ContextMenuItemProps,
+) => React_2.JSX.Element;
+
+// @alpha (undocumented)
+export const contextMenuItemComponentDataRef: ConfigurableExtensionDataRef<
+  ContextMenuItemComponent,
+  'catalog.contextMenuItemComponent',
+  {}
+>;
```

Comment:
We've got a pattern where you can expose these via `dataRefs` of the blueprint instead. Let's move to that if we end up needing this data ref.